apps/taxes/src/modules/taxjar/maps/taxjar-order-created-map.ts

```diff
@@ -0,0 +1,73 @@
+import { LineItem } from "taxjar/dist/types/paramTypes";
+import { CreateOrderRes } from "taxjar/dist/types/returnTypes";
+import { OrderCreatedSubscriptionFragment } from "../../../../generated/graphql";
+import { ChannelConfig } from "../../channels-configuration/channels-config";
+import { CreateOrderResponse } from "../../taxes/tax-provider-webhook";
+import { CreateOrderArgs } from "../taxjar-client";
+import { numbers } from "../../taxes/numbers";
+
+function mapLines(lines: OrderCreatedSubscriptionFragment["lines"]): LineItem[] {
+  return lines.map((line) => ({
+    quantity: line.quantity,
+    unit_price: line.unitPrice.net.amount,
+    product_identifier: line.productSku ?? "",
+    // todo: add from tax code matcher
+    product_tax_code: "",
+    sales_tax: line.totalPrice.tax.amount,
+  }));
+}
+
+export function sumLines(lines: LineItem[]): number {
```

Comment:
same as in other files, avoid exporting such items directly, create a dedicated files for them and wrap them with some helper/util wrappers or keep them private

Reply:
I refactored it, and now it's being exported with the rest of the `taxJarOrderCreated`